ci-operator/REHEARSALS.md

```diff
@@ -0,0 +1,39 @@
+CI job rehearsals
+=================
+
+**This feature is not fully deployed to production yet**
```

Comment:
This needs to be put into words a user will understand -- what is and is not deployed? What works? Maybe just specify what _is_ working

Reply:
addressed